examples/acceptance_tests/039/lib/tests.ak

```diff
@@ -91,16 +91,16 @@ test update_door_angle1() {
 test update_door_locked_and_wheels1() {
   let initial_car =
     Car {
-      owner: #[],
+      owner: #"",
       wheels: 4,
-      vin: #[1, 1, 1, 1, 1, 1, 1],
+      vin: #"01010101010101",
       door: Door { locked: False, hinge_angle: 45 },
     }
   let final_car =
     Car {
-      owner: #[],
+      owner: #"",
       wheels: 5,
-      vin: #[1, 1, 1, 1, 1, 1, 1],
+      vin: #"01010101010101",
```

Comment:
☝️ Makes me think that we might want to add some "aiken fmt" to the CI workflow for all the acceptance tests.